jupyterhub/Chart.yaml

```diff
@@ -9,3 +11,8 @@ sources:
 icon: https://jupyter.org/assets/hublogo.svg
 kubeVersion: '>=1.11.0-0'
 tillerVersion: '>=2.11.0-0'
+maintainers:
+  - name: Erik Sundell
```

Comment:
```suggestion
  # Since it is a requirement of Artifact Hub to have specific maintainers
  # listed, we have added some below, but in practice the entire JupyterHub team
  # contributes to the maintenance of this Helm chart.
  - name: Erik Sundell
```

Reply:
@manics should we go ahead and add this comment and merge?